src/zen_io.c

```diff
@@ -39,14 +39,23 @@
 #define MAX_JSBUF 4096000 // 4MiB
 #endif
 #endif
+
+#if defined(ARCH_CORTEX)
```

Comment:
Create a pull request for revising the use of `ARCH_CORTEX`. Something like `CORTEXM` or `CORTEX_M`.

Reply:
Need to discuss: #160